### ISSUE TITLE
any way to contribute or donate

### ISSUE THREAD

**Issue by foo123** [reduxframework/redux-framework#3177]
without becoming a member (whatever that means)? Would be easier and user does not have to have accounts all over the internet in order to simply contribute to sth.

i have used the plugin in a paid work and would like to give sth back, if you want.

Nikos

**Comment by kmob2** [comment]
Have a look at their official website reduxframework.com, there is a donation button in the footer.

Here is the direct link: https://ideas.reduxframework.com/campaigns/donate-to-keep-redux-alive

**Comment by kmob2** [comment]
Ah, sorry. I didn't realize the donate button will bring you to a sign up page where you have to create an account. I can't really seem to find their paypal address anywhere.

I also just noticed, you can't actually donate on that donate page. I just tried to test donate $10 to see if I could get the paypal infos, but it keeps telling me "Error: Your cart is empty", even though it clearly shows the $10 at the top. I tried multiple times.

**Comment by kprovance** [comment]
What about here:  https://ideas.reduxframework.com/campaigns/donate-to-keep-redux-alive/?_ga=1.12306610.740739422.1401044914

**Comment by kmob2** [comment]
@kprovance that's the same link I posted as well. However there is no way to donate without creating an account on that website, and the other guy was asking to donate without creating an account.

Additionally, I noticed that the link you posted, doesn't actually work. As mentioned above, when you try to donate, and you want to checkout, it always tells you "Error Your cart is empty". In other words, it's not possible to make any donations to your project.
